devel/ci/integration/tests/test_bodhi.py

```diff
@@ -16,16 +16,471 @@
+    query_updates = (
+    # what about build.content, builds, bugs, testcases?
+        "SELECT "
+        "  updates.id as id, "
+        "  updates.alias as alias, "
+        "  updates.notes as notes, "
+        "  updates.status as status, "
+        "  updates.type as type, "
+        "  updates.severity as severity, "
+        "  updates.stable_karma as stable_karma, "
+        "  updates.unstable_karma as unstable_karma, "
+        "  updates.autokarma as autokarma, "
+        "  updates.request as request, "
+        "  updates.locked as locked, "
+        "  updates.suggest as suggest, "
+        "  users.name as username "
+        "FROM updates "
+        "JOIN users ON updates.user_id = users.id "
+        "WHERE status = 'testing' "
+        "ORDER BY date_submitted DESC LIMIT 1"
+    )
+    query_comments = (
+        "SELECT text FROM comments "
+        "WHERE update_id = %s"
+    )
+    db_ip = db_container.get_IPv4s()[0]
+    conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
+    with conn:
+        with conn.cursor() as curs:
+            curs.execute(query_updates)
+            update_info = {}
+            for value, description in zip(curs.fetchone(), curs.description):
+                update_info[description.name] = value
+            curs.execute(query_comments, (update_info["id"], ))
+            expected_comments = []
+            for row in curs.fetchall():
+                expected_comments.append(row[0])
+    conn.close()
+
+    # GET on latest testing update
+    with bodhi_container.http_client(port="8080") as c:
+        headers = {'Accept': 'text/html'}
+        http_response = c.get(f"/updates/{update_info['alias']}", headers=headers)
+
+    try:
+        assert http_response.ok
+        for key, value in update_info.items():
+            if key == "id":
+                pass
+            elif key in ["notes", "request"]:
+                if value != "" and value is not None:
+                    assert value.replace('\n', '') in http_response.text
+            elif key == "stable_karma":
+                assert f"stable threshold: {value}" in http_response.text
+            elif key == "unstable_karma":
+                assert f"unstable threshold: {value}" in http_response.text
+            elif key == "autokarma":
+                if value:
+                    assert "Enabled" in http_response.text
+                else:
+                    assert "Disabled" in http_response.text
+            elif key == "locked":
+                if value:
+                    assert "Locked" in http_response.text
+            elif key == "suggest":
+                if value == "reboot":
+                    assert "Reboot Required" in http_response.text
+                elif value == "logout":
+                    assert "Logout Required" in http_response.text
+            else:
+                assert value in http_response.text
+        # ffmarkdown...
```

Comment:
```suggestion
```

Reply:
At first I wrote the assertions for comments, but I noticed that we sue "markdown" to create links in the middle of comment text. I decided to drop update's comment tests.